### ISSUE TITLE
Update `test` dependency from `nteract-scrapbook` to `scrapbook`

### ISSUE THREAD

**Issue by matthewfeickert** [scikit-hep/pyhf#1725]
### Summary

Running the notebook tests generates the warning

```pytb
warnings.warn("'nteract-scrapbook' package has been renamed to `scrapbook`. No new releases are going out for this old package name.", FutureWarning)
```

as [`nteract-scrapbook`](https://pypi.org/project/nteract-scrapbook/) is now [`scrapbook`](https://pypi.org/project/scrapbook/). All that needs to be done is to change the name used in `steup.py` for the `test` extra:

https://github.com/scikit-hep/pyhf/blob/29bc6daed55b40711fabd9b22d3e76f9ee15657d/setup.py#L42

### Code of Conduct

- [X] I agree to follow the Code of Conduct

**Comment by manulpatel** [comment]
@matthewfeickert Hello! I want to get started to contribute to pyhf. Could you please assign this to me? So that I can quickly fix and close this issue.

**Comment by matthewfeickert** [comment]
Resolved by PR #1841.

**Comment by matthewfeickert** [comment]
Thanks @manulpatel, but it seems that this was an old issue that was already taken care of some time ago. If you want to get involved with development please check out the [`CONTRIBUTING.md`](https://github.com/scikit-hep/pyhf/blob/main/CONTRIBUTING.md). :+1:
